src/main/kotlin/com/sexysisters/tojserverv2/config/SecurityConfiguration.kt

```diff
@@ -22,7 +22,18 @@ class SecurityConfiguration(
     private val jwtTokenProvider: JwtTokenProvider,
     private val jwtValidator: JwtValidator,
 ) {
-    private val VERSION = "/api/v2"
+
+    private val VERSION = "api/v2"
+    private val AUTH = "auth"
+    private val CODE = "code"
+    private val OAUTH = "oauth"
+    private val USER = "user"
+    private val STUDENT = "student"
+    private val TEACHER = "teacher"
+    private val SCHOOL = "school"
+    private val AD = "ad"
+    private val WIKI = "wiki"
+    private val IMAGE = "image"
```

Comment:
질문입니다! 🙋‍♂️
이 부분 `enum`으로 바꿀 수 있을까요?

Reply:
넵 가능할 것 같습니다. 그런데 VERSION을 따로 뺀 이유가 있을까요? 
`$VERSION/auth/code` 보다 `api/v2/auth/code` 를 한꺼번에 관리하는 것도 좋은 것 같습니다! 이건 취향 차이인 것 같아요

Reply:
최대한 중복을 줄이려 한 건데 같이 해줘도 되겠네요
그럼 어떻게 바꿀 수 있을까요?

Reply:
```suggestion

enum class accessUrl(
    val url: String,
) {
    AUTH("/api/v2/auth"),
    USER("/api/v2/user"),
    ...
}
```

이런식으로 관리하면 좋을 것 같아요👍

Reply:
굿입니다 수정할게요!